notebooks/Quickstart/Examples/helloworld.wdl

```diff
@@ -0,0 +1,29 @@
+task HWTask {
+
+  String genome_ref_fasta
+  String version = "hello_world_v1.0.0"
+  String runtime_environment
+
+
+  command {
+    echo ${genome_ref_fasta}
+
+    python -u<<CODE
+
+    chars = """Hello World!"""
+    print(chars)
+
+    import sys
+    print(sys.version)
+
+    CODE
+  }
+
+  runtime {
+    docker: (if runtime_environment == "production" then "python:3.6" else "python:2.7") + "-slim"
+    memory: "100 MB"
```

Comment:
Can we remove these additional runtime params? Since this is just a hello world workflow it doesn't seem necessary

Reply:
To reply to all of the comments on the sample workflows, this sample workflow is moved from elsewhere without a careful modification, I will refactor it so it looks more reasonable!